Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:7.4-cli
+FROM php:8.0-cli
```

Comment:
How did you choose that version? The other container uses 8.1

https://github.com/wikimedia/mediawiki-docker/blob/aa07f326d2719366125551213a2a9be8801aa773/1.39/apache/Dockerfile#L1

I think it would be better to use the same version to avoid additional conflicts at some point in time.